From acd0873d3b3378420fd81dbf68b31f503219e524 Mon Sep 17 00:00:00 2001
From: Nikita Shulga <nshulga@meta.com>
Date: Fri, 23 May 2025 13:04:47 -0700
Subject: [PATCH] [CI] Fix `TestDynamoTimed.test_ir_count` for 3.12 (#154268)

Python-3.12 emits the same bytecode as 3.13 for code in question
Pull Request resolved: https://github.com/pytorch/pytorch/pull/154268
Approved by: https://github.com/clee2000, https://github.com/atalman
ghstack dependencies: #154237
---
 test/dynamo/test_utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/dynamo/test_utils.py b/test/dynamo/test_utils.py
index 595e9dc02fd3..b20713d9ecf4 100644
--- a/test/dynamo/test_utils.py
+++ b/test/dynamo/test_utils.py
@@ -481,7 +481,7 @@ def test_ir_count(self):
             (3, 9): (10, 6),
             (3, 10): (10, 6),
             (3, 11): (10, 6),
-            (3, 12): (10, 6),
+            (3, 12): (11, 7),
             (3, 13): (11, 7),
         }[version]
 
